ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -134,13 +129,7 @@
 import io.confluent.rest.RestConfig;
 import java.io.IOException;
 import java.time.Duration;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Properties;
+import java.util.*;
```

Comment:
nit: I think our guidelines prefer explicit imports

Reply:
Thanks Intellij optimizations hehe.

Reply:
You can disable that in your intellij settings: https://stackoverflow.com/a/3589885/2258040